wrapper/test/output-listener.test.js

```diff
@@ -31,6 +31,6 @@ describe('output-listener', () => {
     listen(Buffer.from('third write'));
     expect(mockWrite.mock.lastCall[0]).toStrictEqual(Buffer.from('third write'));
 
-    expect(mockWrite).toBeCalledTimes(3);
+    expect(mockWrite).toHaveBeenCalledTimes(3);
```

Comment:
Breaking change noted here -> https://jestjs.io/docs/upgrading-to-jest30#removal-of-alias-matcher-functions